roaringbitmap/src/main/java/org/roaringbitmap/BitmapBatchIterator.java

```diff
@@ -0,0 +1,37 @@
+package org.roaringbitmap;
+
+import static java.lang.Long.numberOfTrailingZeros;
+
+public class BitmapBatchIterator implements ContainerBatchIterator {
+
+  private int wordIndex = 0;
+  private long word;
+  private final BitmapContainer bitmap;
+
+  public BitmapBatchIterator(BitmapContainer bitmap) {
+    this.bitmap = bitmap;
+    word = bitmap.bitmap[0];
+  }
+
+  @Override
+  public int next(int key, int[] buffer) {
+    int consumed = 0;
+    while (consumed < buffer.length) {
+      while (word == 0) {
+        ++wordIndex;
+        if (wordIndex == 1024) {
```

Comment:
I don't understand this ``if (wordIndex == 1024)`` clause. Maybe a comment is warranted. How does 1024 relates to ``buffer.length``?

Reply:
Oh. Now I see it.

Reply:
This could indeed be cleaned up.